### DISCUSSION TITLE
Apple/Siri Shortcuts integration

### DISCUSSION THREAD

**Discussion by vinniebearz** [rxhanson/RectanglePro-Community#498] [Ideas]
Being able to trigger Rectangle Pro actions within a shortcut would be pretty cool!

**Comment by rxhanson** [comment]
Thanks for the feedback. At the moment, the interface with Rectangle Pro is via URLs, which you can tie in anywhere:
https://github.com/rxhanson/RectanglePro-Community?tab=readme-ov-file#programmatically-triggering-window-sizes--positions

I'll add Shortcuts integration to my backlog, too, since it would be nice to not have to hunt for the options.
